### DISCUSSION TITLE
X-Axis Skipping Steps and Sometimes Being Loud

### DISCUSSION THREAD

**Discussion by sabaatworld** [dz0ny/klipper-prusa-mk3s#16] [Q&A] [answered]
In prints involving a lot of jerky Y-Axis movements, it seems to skip steps and cause inaccuracies in the print. Subsequent layers are not aligned properly. Another issue that I'm noticing is that during CANCEL_PRINT or other places where X-Axis is to be moved quickly over longer distance, the motor seems to be very noisy.

Do you have suggestions on what things I could try out to fix these issues? I did already confirm that its not an issue with the belt being loose.

Attaching the log file which contains all Klipper configuration. Also recorded a video of this happening.

[klippy.log](https://github.com/dz0ny/klipper-prusa-mk3s/files/10439831/klippy.log)

[Video iCloud Link](https://share.icloud.com/photos/0c3QtM9GAmOvO55mLytyI3DrA)

**Comment by masc2279** [comment, marked as answer]
Set the following

stealthchop_threshold: 0

in both [tmc2130 stepper_y] and [tmc2130 stepper_x]

This should not have been set with a value other than 0. There are known issues with the drivers that cause skipped stepps and bad performance. This is the same for the mks3+ and below.
